README.md

```diff
@@ -181,6 +181,9 @@ python scripts/gr00t_finetune.py --help
 
 # then run the script
 python scripts/gr00t_finetune.py --dataset-path ./demo_data/robot_sim.PickNPlace --num-gpus 1
+
+# run using Lora Parameter Eifficient Fine-Tuning
+python scripts/gr00t_finetune.py  --dataset-path ./demo_data/robot_sim.PickNPlace --num-gpus 1 --lora_rank=32
```

Comment:
i recommend increasing the lora rank in the main readme (to ensure less perf dip)
```
--lora_rank 64  --lora_alpha 128  --batch-size 32
```
also adding a larger batchsize here so user knows they can increase batch-size when using lora.